packages/mirai/lib/src/parsers/mirai_rect_tween/mirai_rect_tween_parser.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:mirai/src/parsers/mirai_rect_tween/mirai_rect_tween.dart';
+import 'package:mirai/src/parsers/mirai_rect/mirai_rect.dart';
+
+class MiraiRectTweenParser {
+  const MiraiRectTweenParser();
+
+  MiraiRectTween getModel(Map<String, dynamic> json) =>
+      MiraiRectTween.fromJson(json);
+
+  RectTween parse(BuildContext context, MiraiRectTween model) {
+    final begin = model.begin?.parse;
+    final end = model.end?.parse;
+
+    switch (model.type) {
+      case 'materialRectArcTween':
+        return MaterialRectArcTween(begin: begin, end: end);
+      case 'materialRectCenterArcTween':
+        return MaterialRectCenterArcTween(begin: begin, end: end);
+      default:
+        return RectTween(begin: begin, end: end);
+    }
+  }
+}
```

Comment:
Can we move this as an `extension` method in `mirai_rect_tweetn.dart`. Just how it's done in [mirai dialog theme](https://github.com/BuildMirai/mirai/blob/db4615de7871161a559922fb2aebf43ce300c346/packages/mirai/lib/src/parsers/mirai_dialog_theme/mirai_dialog_theme.dart#L31)